spring-cloud-previews/google-cloud-compute-spring-starter/src/main/java/com/google/cloud/compute/v1/spring/AcceleratorTypesSpringAutoConfiguration.java

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright 2022 Google LLC
```

Comment:
Noticed the copyright year is still 2022, this is probably a bug in the generator though, similar to [what I did](https://github.com/googleapis/sdk-platform-java/pull/951) last year. Can you take a look?

Reply:
Hmm yeah this part is being handled by the `gapic-generator-java` composers; I've opened https://github.com/googleapis/sdk-platform-java/pull/1720 for the corresponding fix. Thanks for catching!

Reply:
License headers (note: for all generated modules) updated in https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1849/commits/34495a0c768097d5aa430b9a8fe8edc16f2eef21.